shared/common-adapters/swipeable.native.tsx

```diff
@@ -76,61 +76,52 @@ const useSyncClosing = (
 const makePanOnStart = (
   tx: Reanimated.SharedValue<number>,
   startx: Reanimated.SharedValue<number>,
-  dx: Reanimated.SharedValue<number>,
   started: Reanimated.SharedValue<boolean>,
   closeOthersAndRegisterClose: () => void
 ) => {
   return function onStart() {
     closeOthersAndRegisterClose()
-    Reanimated.runOnUI(() => {
-      'worklet'
-      Reanimated.cancelAnimation(tx)
-      startx.value = tx.value
-      dx.value = 0
-      started.value = true
-    })()
+    Reanimated.cancelAnimation(tx)
+    startx.value = tx.value
+    started.value = true
   }
 }
 
 const makePanOnFinalize = (
   tx: Reanimated.SharedValue<number>,
   startx: Reanimated.SharedValue<number>,
-  dx: Reanimated.SharedValue<number>,
   started: Reanimated.SharedValue<boolean>,
   closeSelf: () => void,
   actionWidth: number
 ) => {
-  return function onFinalize(_e: unknown, success: boolean) {
+  return function onFinalize(e: GestureUpdateEvent<PanGestureHandlerEventPayload>, success: boolean) {
     if (!started.value) {
       return
     }
-    const closing = dx.value >= 0
```

Comment:
there are a couple of paths where dx.value is 0 due to some raciness, we can likely just not use this value and use the last event value anyways